### PR TITLE
Chore/remove modules part 6

### PR DESCRIPTION
**Pull request by mimse**
Converted EO bootstrap to standalone

## References

- https://github.com/Energinet-DataHub/greenforce-frontend/issues/1744

### CODE REVIEW

**Review by dzhavat — COMMENTED** [review]
praise: looks nice 💯

note: this is review only for the changes under the "gf" folder.

File: libs/gf/util-browser/src/lib/gf-browser-configuration.module.spec.ts [hunk 1]
```diff
@@ -17,31 +17,23 @@
...
   it('APP_BASE_HREF is not provided when the Angular module is not imported', () => {
-    const actualAppBaseHref = TestBed.inject(APP_BASE_HREF, null);
+    const actualAppBaseHref = TestBed.inject(APP_BASE_HREF, null, {});
     expect(actualAppBaseHref).toBeNull();
   });
 
   it('APP_BASE_HREF is provided when the Angular module is imported', () => {
     TestBed.configureTestingModule({
-      imports: [GfBrowserConfigurationModule.forRoot()],
+      imports: [NoopAnimationsModule, RouterTestingModule],
```
[the remark this answers is not in the document]
Comment (MadsThy) [reply libs/gf/util-browser/src/lib/gf-browser-configuration.module.spec.ts:32]: Removed

File: libs/gf/util-browser/src/lib/gf-browser-configuration.module.spec.ts [hunk 2]
```diff
@@ -17,31 +17,23 @@
...
-describe(GfBrowserConfigurationModule.name, () => {
+describe('browserConfigurationProviders', () => {
   it('APP_BASE_HREF is not provided when the Angular module is not imported', () => {
-    const actualAppBaseHref = TestBed.inject(APP_BASE_HREF, null);
+    const actualAppBaseHref = TestBed.inject(APP_BASE_HREF, null, {});
     expect(actualAppBaseHref).toBeNull();
   });
 
   it('APP_BASE_HREF is provided when the Angular module is imported', () => {
     TestBed.configureTestingModule({
```
[the remark this answers is not in the document]
Comment (MadsThy) [reply libs/gf/util-browser/src/lib/gf-browser-configuration.module.spec.ts:31]: Got rid of the text :)

**Review by MadsThy — APPROVED** [review]
LGTM

**Pull request merged**